docs/develop/add-ins-with-angular2.md

```diff
@@ -89,7 +89,7 @@ The Office Add-in dialog API enables your add-in to open a page in a nonmodal di
 
 The [displayDialogAsync](/javascript/api/office/office.ui) method takes a parameter that specifies the URL of the page that should open in the dialog box. Your add-in can have a separate HTML page (different from the base page) to pass to this parameter, or you can pass the URL of a route in your Angular application. 
 
-It is important to remember, if you pass a route, that the dialog box creates a new window with its own execution context. Your base page and all its initialization and bootstrapping code run again in this new context, and any variables are set to their initial values in the dialog box. So this technique launches a second instance of your single page application in the dialog box. Code that changes variables in the dialog box does not change the task pane version of the same variables. Similarly, the dialog box has its own session storage, which is not accessible from code in the task pane.  
+It is important to remember, if you pass a route, that the dialog box creates a new window with its own execution context. Your base page and all its initialization and bootstrapping code run again in this new context, and any variables are set to their initial values in the dialog box. So this technique launches a second instance of your single page application in the dialog box. Code that changes variables in the dialog box does not change the task pane version of the same variables. Similarly, the dialog box has its own session storage (that is, the [Window.sessionStorage](https://developer.mozilla.org/docs/Web/API/Window/sessionStorage) property), which is not accessible from code in the task pane.  
```

Comment:
Could we get away with not including "that is"? I think the callout works equally well without it and it reads a little cleaner.

I'd also love to have "session storage" just link off to the Mozilla docs (like "the dialog box has its own [session storage](https://developer.mozilla.org/docs/Web/API/Window/sessionStorage)"), but that might be a bit unclear.

Reply:
@ElizabethSamuel-MSFT and @AlexJerabek Something really bizarre is happening with GitHub today. Those metadata dates were changed to today's date in the very first commit of this PR. Also, I already changed the ";" after "that is" to a comma. 

The latter problem will be moot because I'll take Alex's suggestion to just get rid of the "that is"; but I don't know why Elizabeth is seeing the old dates. I don't know how to fix that.